### ISSUE TITLE
Plugin not supports field reservation

### ISSUE THREAD

**Issue by ShuffleBoy** [devkanro/intellij-protobuf-plugin#227]
1) Create a proto file and paste
```proto
syntax = "proto3";

message Test{
    reserved 2;
    reserved "to_be_reserved";
    
    int32 field = 1;
    int32 to_be_reserved = 3;
}
```

2) The plugin doesn't recognize field reservations (as mentioned in language guide https://protobuf.dev/programming-guides/proto3/#fieldreserved)
![изображение](https://github.com/devkanro/intellij-protobuf-plugin/assets/44951140/09822af7-a414-4935-84c7-1923f2c9d755)

**Comment by devkanro** [comment]
Currently we support those symbol:
```protobuf
syntax = "proto3";

message Test{
    reserved 2;
    reserved to_be_reserved;

    int32 field = 1;
    int32 to_be_reserved = 3;
}
```

I will update the AST to support this.

**Comment by devkanro** [comment]
Will be resolved in 2.0.0, currently we are running [2.0.0-EAP](https://github.com/devkanro/intellij-protobuf-plugin/discussions/253), you can join us to test it.
